guestlib/queue_worker.h

```diff
@@ -0,0 +1,21 @@
+#ifndef _AVA_GUESTLIB_QUEUE_WORKER_H_
+#define _AVA_GUESTLIB_QUEUE_WORKER_H_
+
+#include "common/cmd_channel.hpp"
+#include "common/declaration.h"
+#include "common/linkage.h"
+
+namespace ava {
+
+class EXPORTED QueueWorker {
```

Comment:
Should it be `CmdQueue`  and define in `common/cmd_queue.h`?
(API server may use the same interface to implement reply command queue in the future.)